components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/TokenBindingMgtDAO.java

```diff
@@ -76,4 +76,17 @@ default Optional<TokenBinding> getTokenBindingByBindingRef(String tokenId, Strin
      * @throws IdentityOAuth2Exception in case of failure.
      */
     void deleteTokenBinding(String tokenId) throws IdentityOAuth2Exception;
+
+    /**
+     * Obtain token binding from refresh token.
+     *
+     * @param refreshToken Refresh token sent in the request.
+     * @param isTokenHashingEnabled Whether token hashing is enabled.
```

Comment:
Lets add the retun in the comment

Reply:
Fixed in 352953d9b07be7e4c728d94e03332a371ba649b3